app.js

```diff
@@ -1,46 +1,110 @@
 let fs = require('fs'),
     mysql = require('mysql'),
-    parse = require('csv-parse');
+    parse = require('csv-parse'),
+    filesToInsert,
+    isUserSpecified = false;
+
 
 const DB_CONFIG = require('./dbconfig.json');
+var connection = mysql.createConnection(DB_CONFIG);
+
 
 // function makes any unvaild csv's name valid
 function convertFileNameToTableName(file) {
     file = file.slice(0, -4).replace(/`/g, "``");
-
     return "`" + file + "`";
 }
 
-// Enumerate all files inside the "files" folder
-fs.readdir('./files/', 'utf8', (error, files) => {
-    if (error) {
-        console.error(error);
-        return;
-    }
+function CreateTable(createTableQuery, parsedData, tableName, insertRows) {
+    connection.query(createTableQuery, error => {
+        if (error) {
+            console.error(error);
+            return;
+        }
+        insertRows(tableName, parsedData);
+    });
+}
 
-    // Filter csv files
-    let csvFiles = files.filter(file => /(\.csv)$/.test(file));
-    if (!csvFiles.length) {
-        console.error(new Error('No CSV files found'));
-        return;
-    }
+function insertRows(tableName, parsedData) {
+    let completedQueryCount = 0;
+    parsedData.forEach((row, index, array) => {
+        let rowToInsert = [];
```

Comment:
This code will be unnecessary if you use parse without {columns:true} because each row will already be an array of values